types/underscore/index.d.ts

```diff
@@ -5255,7 +5255,7 @@ declare module _ {
         * Wrapped type `any[]`.
         * @see _.invoke
         **/
-        invoke(methodName: string, ...args: any[]): _Chain<T>;
+        invoke(methodName: string, ...args: any[]): _Chain<T, V>;
```

Comment:
This is actually unknown as you don't know the return value of the method

Reply:
Changed to _Chain<any, any[]>